### PR TITLE
Add email-templates

### PR DESCRIPTION
**Pull request by killmenot**
**By submitting this pull request, I promise I have read the [contribution guidelines](https://github.com/sindresorhus/awesome-nodejs/blob/master/contributing.md) twice and ensured my submission follows it. I realize not doing so wastes the maintainers' time that they could have spent making the world better. 🖖**

⬆⬆⬆⬆⬆⬆⬆⬆⬆⬆

### CODE REVIEW

**Comment by sindresorhus** [comment]
> Don't mention Node.js in the description as it's implied.

And the description could use some improvements in general.

// @niftylettuce

**Comment by niftylettuce** [comment]
I'm about to release v3.0.0.

@killmenot can you please change it to read:

```md
[email-templates](https://github.com/niftylettuce/node-email-templates) - Create, preview, and send custom email templates for Node.js. Highly configurable and supports automatic inline CSS, stylesheets, embedded images and fonts, and much more!
```

**Comment by sindresorhus** [comment]
@niftylettuce That's too long. Descriptions here should be short. See the other descriptions.

**Comment by niftylettuce** [comment]
This is probably best then:

> Create, preview, and send custom email templates for Node.js.

Do you agree?

**Comment by niftylettuce** [comment]
I didn't want to over complicate it or talk about how overly configurable it is; people might get scared?

**Comment by sindresorhus** [comment]
> Don't mention Node.js in the description as it's implied.

Looks good, but not :arrow_up:

**Comment by niftylettuce** [comment]
> Create, preview, and send custom email templates

do you think it's good enough like that @sindresorhus?

curious if you had some other creative-take on the description 🎉 🤤

**Comment by killmenot** [comment]
@sindresorhus Absolutely, have no anything against 
> Don't mention Node.js in the description as it's implied.

but node.js mentioned really many times in readme.md. It's the only reason why I kept original author's description.

**Comment by killmenot** [comment]
@niftylettuce will update PR once @sindresorhus approve the description. I think "Create, preview, and send custom email templates" is good one

**Comment by sindresorhus** [comment]
> but node.js mentioned really many times in readme.md. It's the only reason why I kept original author's description.

Yes, sometimes it's necessary to mention it, like when talking about something being supported in both the browser and Node.js. For example, it makes sense here: "Rapid prototyping framework for making mock HTTP APIs, with a Node.js, command-line and HTTP interface." General rule is, if the sentence still makes sense when you remove it, it should be removed.

**Comment by sindresorhus** [comment]
> Create, preview, and send custom email templates

👍

File: readme.md [hunk 1]
```diff
@@ -674,6 +674,7 @@ Just type [`node.cool`](https://node.cool) to go here ✨
 
 - [Nodemailer](https://github.com/andris9/Nodemailer) - The fastest way to handle email.
 - [emailjs](https://github.com/eleith/emailjs) - Send text/HTML emails with attachments to any SMTP server.
+- [Node Email Templates](https://github.com/niftylettuce/node-email-templates) - Node.js module for rendering beautiful emails with ejs templates and email-friendly inline CSS using juice.
```
Comment (sindresorhus) [inline readme.md]: The name and URL also needs to be updated.

**Comment by killmenot** [comment]
@sindresorhus updated PR

**Pull request merged**